utils_cv/similarity/model.py

```diff
@@ -107,34 +108,40 @@ def compute_features_learner(
     Returns: DNN feature of the provided image.
 
     """
-    if dataset_type == DatasetType.Train:
+    # Note: In Fastai, for DatasetType.Train, only the output of complete minibatches is computed. Ie if one has 101 images, 
+    # and uses a minibatch size of 10, then len(feats) is 96 and not 101. For DatasetType.Valid this is not the case,
```

Comment:
"...and uses a minibatch size of 10, then len(feats) is 96 and not 101." should that "10" be a 4, 8 or 16 or something that is divisible into 96

Reply:
Thanks. the 10 was a typo, changed to 16.